src/views/product-root-docs-path-landing/index.tsx

```diff
@@ -34,7 +34,7 @@ const ProductRootDocsPathLanding = ({
 	}
 
 	return (
-		<div className={s.root}>
```

Comment:
This was only added for the version selector so a class is not needed here otherwise